Blockzilla/AppShortcuts/ShortcutView.swift

```diff
@@ -100,6 +100,16 @@ class ShortcutView: UIView {
         delegate?.shortcutTapped(shortcut: shortcut)
     }
     
+    func getShortcut() -> Shortcut {
+        return shortcut
+    }
+    
+    func renameShortcut(newName: String) {
```

Comment:
I think we should pass and change the shortcut model in the view as well. This method cand be something like `func update(with shortcut: Shortcut)`